src/Microsoft.Framework.Configuration.File/Helper/ConfigurationHelper.cs

```diff
@@ -10,18 +10,13 @@ public static class ConfigurationHelper
     {
         public static string ResolveConfigurationFilePath(IConfigurationBuilder configuration, string path)
```

Comment:
I am ok with this, although I am not sure about abbreviating `Configuration` (I am not sure we have so far done it) and I still would prefer to have `Resolve` in the name.